Plugin 'UELlama' failed to load because module 'UELlama' could not be loaded

I'm having issues implementing the plugin. I've come from using the original UELlama plugin which worked but I had issues with packaging the project. I'm pretty novice and I'm a bit confused on things.

I'm running the Oculus source build UE5.3 on Windows and when I try to follow the instructions to build using cmake I get the error CMakeLists.txt doesn't exist.
So I build through the Visual Studio IDE but when I try to run UE I get:
- Plugin 'UELlama' failed to load because module 'UELlama' could not be loaded.  There may be an operating system error or the module may not be properly set up.

> What version of Cuda are you using, I have a feeling im not on the right version

Ive gotta go to work now :) Night shift. I will try the CPU one tomorrow ASAP. The best error I got so far is : 
TLDR , dont think Im getting a .lib file somewhere
C:\Users\skyog\Documents\GitHub\ToTheMoonReLlama\Binaries\Win64\UnrealEditor-ToTheMoon.patch_1.lib and object C:\Users\skyog\Documents\GitHub\ToTheMoonReLlama\Binaries\Win64\UnrealEditor-ToTheMoon.patch_1.exp
llama.lib(ggml.obj) : error LNK2019: unresolved external symbol __imp_strdup referenced in function gguf_add_tensor
C:\Users\skyog\Documents\GitHub\ToTheMoonReLlama\Binaries\Win64\UnrealEditor-ToTheMoon.patch_1.exe : fatal error LNK1120: 1 unresolved externals.

--- a/Source/LlamaCore/LlamaCore.Build.cs
+++ b/Source/LlamaCore/LlamaCore.Build.cs
@@ -145,8 +145,8 @@ public class LlamaCore : ModuleRules
 			//temp
 			string WinLibDLLPath = LlamaPath;
 
-			PublicDelayLoadDLLs.Add("ggml.dll");
-			PublicDelayLoadDLLs.Add("llama.dll");
+			// PublicDelayLoadDLLs.Add("ggml.dll");
+			// PublicDelayLoadDLLs.Add("llama.dll");
 
 			RuntimeDependencies.Add("$(BinaryOutputDir)/ggml.dll", Path.Combine(WinLibDLLPath, "ggml.dll"));
 			RuntimeDependencies.Add("$(BinaryOutputDir)/llama.dll", Path.Combine(WinLibDLLPath, "llama.dll"));

--- a/Source/LlamaCore/Private/LlamaComponent.cpp
+++ b/Source/LlamaCore/Private/LlamaComponent.cpp
@@ -802,19 +802,21 @@ ULlamaComponent::ULlamaComponent(const FObjectInitializer &ObjectInitializer)
                     {
                         ModelState.LastRole = LastRoleFromStructuredHistory();
                     }
-
                     //Grab partial from last message
-                    const FStructuredChatMessage& Message = ModelState.ChatHistory.History.Last();
-
-                    //Confirm it's from the assistant
-                    if (Message.Role == EChatTemplateRole::Assistant)
+                    
+                    if(ModelState.ChatHistory.History.Num() > 0)
                     {
-                        //Look for period preceding this one
-                        FString Sentence = GetLastSentence(Message.Content);
-
-                        if (!Sentence.IsEmpty())
+                        const FStructuredChatMessage &Message = ModelState.ChatHistory.History.Last();
+                        //Confirm it's from the assistant
+                        if (Message.Role == EChatTemplateRole::Assistant)
                         {
-                            OnPartialParsed.Broadcast(Sentence);
+                            //Look for period preceding this one
+                            FString Sentence = GetLastSentence(Message.Content);
+
+                            if (!Sentence.IsEmpty())
+                            {
+                                OnPartialParsed.Broadcast(Sentence);
+                            }
                         }
                     }
                 }

--- a/Source/LlamaCore/Public/LlamaComponent.h
+++ b/Source/LlamaCore/Public/LlamaComponent.h
@@ -170,7 +170,7 @@ struct FLLMModelParams
     UPROPERTY(EditAnywhere, BlueprintReadWrite, Category = "LLM Model Params")
     FString PathToModel = "./model.gguf";
 
-    UPROPERTY(EditAnywhere, BlueprintReadWrite, Category = "LLM Model Params")
+    UPROPERTY(EditAnywhere, BlueprintReadWrite, Category = "LLM Model Params", meta=(MultiLine=true))
     FString Prompt = "You are a helpful assistant.";
 
     //If not different than default empty, no template will be applied
@@ -287,7 +287,7 @@ public:
     TMap<FString, FChatTemplate> CommonChatTemplates;
 
     UFUNCTION(BlueprintCallable, Category = "LLM Model Component")
-    void InsertPrompt(const FString &Text);
+    void InsertPrompt(UPARAM(meta=(MultiLine=true)) const FString &Text);
 
     /** 
     * Use this function to bypass input from AI, e.g. streaming input from another source. 
@@ -304,7 +304,7 @@ public:
 
     //This will wrap your input given the specific role using chat template specified
     UFUNCTION(BlueprintCallable, Category = "LLM Model Component")
-    void InsertPromptTemplated(const FString& Text, EChatTemplateRole Role);
+    void InsertPromptTemplated(UPARAM(meta=(MultiLine=true)) const FString& Text, EChatTemplateRole Role);
 
 
     UFUNCTION(BlueprintCallable, Category = "LLM Model Component")

